webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -4025,6 +4025,7 @@ var KM_ATTACHMENT_V2_SUPPORTED_MIME_TYPES = ["application","text","image"];
                         CURRENT_GROUP_DATA.conversationAssignee = data && data.groupFeeds[0] && data.groupFeeds[0].metadata.CONVERSATION_ASSIGNEE;
                         CURRENT_GROUP_DATA.groupMembers = data.userDetails && data.userDetails;
                         CURRENT_GROUP_DATA.lastMessagingMember = data.message[0] && data.message[0].contactIds;
+                        params.isWaitingQueue && KommunicateUI.handleWaitingQueueMessage();
```

Comment:
@Kusum-Dhoundiyal Can you check one case here?

When socket connection is slow, we're fetching conversation messages from message list API.
In that case, "viaCreateGroup" value will be true. So isWaitingQueue value should be true in that case also I suppose.

Reply:
@ab14bhardwaj  with slow connection we tried it was working fine.